Client/src/main/java/com/lnc/controller/LoginController.java

```diff
@@ -0,0 +1,91 @@
+package com.lnc.controller;
+
+import com.lnc.service.AuthService;
+import com.lnc.util.InputHandler;
+
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class LoginController {
+  AuthService auth = new AuthService();
+
+  public void loginUser() throws Exception {
+    String employeeID = getEmployeeID();
+    String password = getPassword();
+    String hashPassword = getHashPassword(password);
```

Comment:
controller should not have logics, it should just accept the input, validate & pass it to service